src/pages/Locations/LocationItem.css

```diff
@@ -0,0 +1,53 @@
+.custom-list {
+  background-color: #f4f4f4;
+}
+
+.custom-item {
+  border-bottom: 1px solid #ddd;
+  cursor: pointer;
+}
+
+.ion-text-micro-wrap {
```

Comment:
.ion-* classes are reserved and used by ionic and we can import directly from ionic and shouldn't redefine them.